script/libraries/Constants.sol

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8;
+
+library Constants {
+    address constant CREATE2_FACTORY = 0x700b6A60ce7EaaEA56F065753d8dcB9653dbAD35;
```

Comment:
I had a different address while running this locally yday - is this the randamu one or the foundry one or something else?
You could possibly move it to the .env file seeing as it's only used in the deployment scripts